subsys/mgmt/mcumgr/grp/img_mgmt_client/src/img_mgmt_client.c

```diff
@@ -266,7 +266,8 @@ static size_t upload_message_header_size(struct img_gr_upload *upload_state)
 	} else {
 		map_count = 10;
 	}
-
+	
+	temp_data = NULL;
```

Comment:
Move to line 258 as part of the variable setup

Reply:
Thanks, I moved it to line 258.